.azure-pipelines/pipelines.yml

```diff
@@ -337,7 +337,7 @@ stages:
               itemPattern: "docker/envoy-docker-images.tar.xz"
               downloadType: single
               targetPath: $(Build.StagingDirectory)
-          - bash: ./ci/do_ci.sh verify_examples
+          - bash: ./ci/run_envoy_docker.sh './ci/do_ci.sh verify_examples'
```

Comment:
?

Reply:
as i understand it, its preferred not to apt-get things in ci - so i added expect to the ci/build image previously

Reply:
Ah OK I see. Sounds good.

Reply:
reverted